tests/chaos/oadp/test_oadp.py

```
@@ -68,3 +73,118 @@ def test_drain_vm_node_during_backup(
     oadp_backup_in_progress.wait_for_status(
         status=oadp_backup_in_progress.Backup.Status.COMPLETED, timeout=TIMEOUT_10MIN
     )
+
+
+@pytest.mark.chaos
+@pytest.mark.parametrize(
+    "pod_deleting_process_during_oadp_operations, expected_status",
```

Ruff (0.14.14)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)